corehq/apps/reports/standard/project_health.py

```diff
@@ -184,12 +205,27 @@ def show_in_navigation(cls, domain=None, project=None, user=None):
     def decorator_dispatcher(self, request, *args, **kwargs):
         super(ProjectHealthDashboard, self).decorator_dispatcher(request, *args, **kwargs)
 
-    @property
-    def template_context(self):
+    def get_filtered_group_ids(self):
+        return self.request.GET.getlist('group')
+
+    def get_users_by_filtered_groupids(self):
+        groupids_param = self.get_filtered_group_ids()
+        users_list = GroupES().domain(self.domain).group_ids(groupids_param).source(["users"]).values()
```

Comment:
I think you can use `values_list` here instead and skip the indexing below:
`users_list = GroupES().domain(self.domain).group_ids(groupids_param).values_list("users")`